### PR TITLE
Portfolio edits branch four

### PR DESCRIPTION
**Pull request by EmmaWenzel**
Some final touches with CSS styling

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -30,3 +30,21 @@ function addRandomFact() {
...
 }
 
+/** 
+    Sets tab buttons display content 
+*/
+function openTab(tab) {
+    let tabcontent;
+
+    // Hide each tab 
+    tabcontent = document.getElementsByClassName("tabcontent"); // get each tab labeled tab content
```
Comment (ofk-goog) [inline portfolio/src/main/webapp/script.js:40]: The "Hide each tab" comment is good. It concisely explains the following block of code.

But I'd remove the "get each tab labeled tab content" comment, which doesn't add much.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -30,3 +30,21 @@ function addRandomFact() {
   factContainer.innerText = fact;
 }
 
+/** 
+    Sets tab buttons display content
```
Comment (ofk-goog) [inline portfolio/src/main/webapp/script.js:34]: It's great to document functions this way. But this description is hard to understand. Maybe something like "Switches the view to the given tab." WDYT?

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,9 +1,37 @@
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  margin: auto;
+  width: 1000px;
+  font-family: impact, black, serif;
```
[the remark this answers is not in the document]
Comment (EmmaWenzel) [reply portfolio/src/main/webapp/style.css:4]: Any other font suggestions? I could switch back to Times New Roman it's always a classic but I had trouble coming up with something else on google

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (EmmaWenzel) [reply portfolio/src/main/webapp/script.js]: Rereading that description it seems like it's barely English, thanks! I switched it to "Shows the content of the active tab"

File: portfolio/src/main/webapp/style.css (hunk 3 above)
[the remark this answers is not in the document]
Comment (ofk-goog) [reply portfolio/src/main/webapp/style.css:4]: I don't know fonts either. Do what you think is right.

**Pull request merged**